pinot-common/src/main/proto/plan.proto

```diff
@@ -193,3 +194,29 @@ message WindowNode {
   int32 upperBound = 6;
   repeated Literal constants = 7;
 }
+
+message ExplainNode {
+  string type = 1;
```

Comment:
Yep, this is basically the _type_ of operator, which may be anything. Remember that these operators can even be extended with plugable indexes. Anyway we can add a comment to make it clearer for future readers :+1:

Reply:
I think I'm going to rename it as _title_. WDYT?

Reply:
I think title is a better name. Type seems to indicate that it is a unique thing that completely identifies the operator, when in fact it may not be. I've added some documentation which emphasizes that explain nodes should not be used to reconstruct the operators.